app/assets/scripts/actions/index.js

```diff
@@ -26,6 +28,35 @@ export function createFieldReport (payload) {
   return postJSON('api/v1/field_report/', CREATE_FIELD_REPORT, payload, withToken());
 }
 
+export const GET_SURGE_ALERTS = 'GET_SURGE_ALERTS';
+export function getSurgeAlerts (page = 1, filters = {}) {
+  filters.limit = filters.limit || 10;
+  filters.offset = filters.limit * (page - 1);
+  const f = buildAPIQS(filters);
+
+  return fetchJSON(`/api/v1/surge_alert/?${f}`, GET_SURGE_ALERTS, withToken());
+}
+
+export const GET_APPEALS_LIST = 'GET_APPEALS_LIST';
+export function getAppealsList () {
+  const f = buildAPIQS({
+    end_date__gt: DateTime.local().toISODate(),
+    limit: 100
```

Comment:
What's your plan for this, and the recursive function below, with real data? I feel like setting the limit to `0` should be ok now that we're using reasonable development data?

Reply:
It's just a matter of removing the recursion block.